js/core/dropdown/views/PopoutView.js

```diff
@@ -295,15 +295,15 @@ export default Marionette.LayoutView.extend(/** @lends module:core.dropdown.view
         if (this.currentDirection === popoutDirection.UP || anchorBottomOffset < panelHeight) {
             this.currentDirection = popoutDirection.UP;
             this.panelRegion.$el.css({
-                top: -panelHeight
+                top: (parseInt(anchor.css('top')) || 0) - panelHeight
```

Comment:
The value will be invalid if anchor is not the direct child of the button. It's better to compute the offset this way: `anchor.getBoundingClientRect().top - button.getBoundingClientRect().top`